### DISCUSSION TITLE
Clarity on RustDesk Pro plans

### DISCUSSION THREAD

**Discussion by Jono-Moss** [rustdesk/rustdesk#5206] [Q&A] [answered]
Good Day all,

I have been a sponsor for a few months (Thanks for the awesome software) and have only noticed now that you have added the Pro services. 
I'm very excited for the "Address book" feature. 
But I'm a little confused about how the "users" limit works.
Let's say that I'm the only person who will need the "Address book" or any pro features.
I can get the "Personal Plan" and can have up to 100 "Saved contacts" in the address book.
But then for example, the 100 contacts I have, they do not need a license as the license is only needed for
the people who need the "Pro" features.
And the 100 contacts on my "relay server" can continue to use RustDesk to connect to each other without a license?
Or will the "Personal Plan" only allow me to have a total of 100 users on my "relay server".

So to try summarize,
does the "user limit" affect the number of devices I can "Relay" / connect to my server.
Or is the "user Limit" only limit the number of users I can save in my "Address Book" (Managed Devices)? 

I hope this makes sense. 
Thank you.

**Comment by rustdesk** [comment, marked as answer]
> does the "user limit" affect the number of devices I can "Relay" / connect to my server

No limit, Pro is based on Open source version, it only add features on Open source version, won't degrade from Open source version.

**Reply by Jono-Moss** [reply]
Wow, thank you very much.
I'm definitely looking in to upgrading soon.
Thank you very much for all your hard work.
All the best.
